clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -978,9 +978,9 @@ public static void atomicMoveWithFallback(Path source, Path target, boolean need
             Files.move(source, target, StandardCopyOption.ATOMIC_MOVE);
         } catch (IOException outer) {
             try {
+                log.warn("Failed atomic move of {} to {} retring with a non-atomic move", source, target, outer);
                 Files.move(source, target, StandardCopyOption.REPLACE_EXISTING);
-                log.debug("Non-atomic move of {} to {} succeeded after atomic move failed due to {}", source, target,
-                        outer.getMessage());
+                log.debug("Non-atomic move of {} to {} succeeded after atomic move failed", source, target);
```

Comment:
Remain DEBUG?

Reply:
Maybe it is okay now that we are logging the initial WARn?

Reply:
Yeah. I think so. I care more about the atomic move failing and less about the non-atomic move succeeding.